setup.cfg

```diff
@@ -23,6 +23,7 @@ setup_requires = setuptools_scm[toml] >= 3.4.1
 [options.extras_require]
 testing =
 	# upstream
+	coverage < 5
```

Comment:
Nit: I'd prefer this to appear at the end of the list - that's where I typically add them.